### DISCUSSION TITLE
v1.5.0 - Release Candidate 1

### DISCUSSION THREAD

**Discussion by bohdanbobrowski** [bohdanbobrowski/blog2epub#29] [Announcements]
- [X] integration testing
- [X] increase unit test coverage
- [X] use sitemaps.xml for scraping
- [X] crawlers refactor
  - [X] use data models
  - [X] more common methods in crawler class
  - [X] expand crawler abstract
- [X] cli interface refactor
- [X] greek alphabet support
- [X] image download and attachment bug solved (ex. modernistyczny-poznan.blogspot.com)
- [X] improved resistance to http errors
- [X] dedicated crawler class for zeissikonveb.de
- [X] (on GUI) skip value is enlarged on limit value (if such is set)
- [X] download progress is much more verbose, also on GUI it can be cancelled everytime

<hr /><em>This discussion was created from the release <a href='https://github.com/bohdanbobrowski/blog2epub/releases/tag/'>v1.5.0 - Release Candidate 1</a>.</em>

**Comment by camifran** [comment]
Hi! Could you provide a Mac compiled version?

thank you!!!
